### PR TITLE
docs(docker source): Refer to docker engine api

### PR DESCRIPTION
**Pull request by LucioFranco**
Signed-off-by: Lucio Fanco <luciofranco14@gmail.com>

### CODE REVIEW

File: .meta/sources/docker.toml [hunk 1]
```diff
@@ -4,10 +4,10 @@ beta = true
 common = true
 delivery_guarantee = "best_effort"
 function_category = "collect"
-min_version = "1.24"
```
Comment (binarylogic) [inline .meta/sources/docker.toml]: Why was this removed?

Comment (LucioFranco) [reply .meta/sources/docker.toml]: There are many versions when you call the docker version command. I updated this to clarify exactly what the requirement is for.

**Comment by binarylogic** [comment]
Superceded by https://github.com/timberio/vector/pull/1885
